### PR TITLE
Added support for OverridePCC tag

### PR DESCRIPTION
**Pull request by jirismika**
Hello, 
I've added the OverridePCC tag ( #50 ) to all AirTemplates.

### CODE REVIEW

**Comment by shmuga** [comment]
Awesome work @jirismika!
I have just requested a review from @dchertousov and after that, we'll merge changes 😎

File: README.md [hunk 1]
```diff
@@ -72,15 +72,16 @@ It also has several useful helpers to handle errors.
 
 | Param | Type | Default | Description |
 | --- | --- | --- | --- |
-| auth | `Object <username, password, targetBranch>` | - | See `auth` description [below](#auth). |
+| auth | `Object <username, password, targetBranch, pcc, overridePcc>` | - | See `auth` description [below](#auth). |
```
Comment (dchertousov) [inline README.md]: I see two new params added in docs: `pcc` and `overridePcc`.
I can see what is `overridePcc` responsible for, but what about `pcc`?
I found it's only used in Air service and only in this line:
https://github.com/Travelport-Ukraine/uapi-json/blob/master/src/Services/Air/AirService.js#L25
(and this variable is used then here: https://github.com/Travelport-Ukraine/uapi-json/blob/master/src/Services/Air/templates/AIR_CREATE_RESERVATION_REQUEST.xml#L53)

Do you think, guys that `pcc` should really be passed in `auth`?
I can see clear reasons why `overridePcc ` should but not `pcc`.
@shmuga , @jirismika ?

Comment (articice) [inline README.md]: @dchertousov it is the effective pcc of credentials being used. It is copied into each of the search results and not used anywhere else; it is a sort of previous design artefact.
We can remove it, then users should control which pcc they're using by external logic, which fits in perfectly with overridePcc.

**Comment by Smotrov** [comment]
As far as uAPI JSON is made to simplify understanding, my suggestion is to use _emulatePcc_ instead of _overridepcc_. Because is common phrase in industru _to emulate some ones PCC_, but no _to override some ones PCC_.

**Comment by dchertousov** [comment]
@Smotrov 
In my opinion emulating someone's PCC is the same as overriding your own.
In terms of GD SEM command is actually emulating someone's screen, but do we,?

File: README.md (hunk 1 above)
Comment (dchertousov) [inline README.md]: If it's used only for search results, I think we have to remove it, as any user of library takes control of what's going on.

**Comment by Smotrov** [comment]
Please use only one parameter. Call it **emulatePcc**. Obviously it shouldn't be mandatory.

**Comment by dchertousov** [comment]
@jirismika any updates on `emulatePcc` parameter?

**Comment by dchertousov** [comment]
I'm going to merge this to master as it is with `overridePcc` name, and then change it to `emulatePcc``

**Pull request merged**